ftw/file/locales/de/LC_MESSAGES/ftw.file.po

```diff
@@ -93,6 +93,11 @@ msgstr "Die existierende Datei wird beim Upload überschrieben.<br />Drücken Si
 msgid "download"
 msgstr "herunterladen"
 
+#. Default: "Download not possible because the file contains a virus (${name})."
+#: ./ftw/file/fields.py:99
+msgid "download_not_possible"
+msgstr ""
```

Comment:
German translation:
Download blockiert. In der Datei wurde die Schadsoftware ${name} gefunden.

Reply:
done